letsdrill.rb

```diff
@@ -1,13 +1,22 @@
 def get_letter_grade(integer)
  
-  #Put your code here!
-
+  grade = case integer
+    when 0..59 then "F"
+    when 60..69 then "D"
+    when 70..79 then "C"
+    when 80..89 then "B"
+    when 90..100 then "A"
+    else "Invalid Grade"
+    end
 end
 
 def shortest_string(array)
-
-  #Put your code here!
-
+  return nil if array.length == 0
```

Comment:
good effort! i think you could refactor your code here. would something like the following work?
http://ruby-doc.org/core-2.4.0/Enumerable.html#method-i-min_by